UltimateMedals.as

```diff
@@ -173,14 +146,17 @@ void Render() {
 				// We don't care about the max length, so we render it normally
 				UI::Text(mapNameText);
 			}
+
+			if (hasComment && UI::BeginItemTooltip()) {
+				UI::PushTextWrapPos(200);
+				UI::TextWrapped(map.MapInfo.Comments);
+				UI::PopTextWrapPos();
+				UI::EndTooltip();
+			}
 		}
 
 		if(showAuthorName) {
-			string authorNameText = "\\$888by " + Text::StripFormatCodes(map.MapInfo.AuthorNickName);
-			if (hasComment) {
-				authorNameText += " \\$68f" + Icons::InfoCircle;

```

Comment:
It seems like the 'i' icon is removed. If intentional, should probably update the `showComment` setting description to match.